Sources/MapboxNavigation/UIViewController.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+extension UIViewController {
+    
+    func embed(_ viewController: UIViewController, in view: UIView) {
```

Comment:
This is handy; surprised we didn’t have something like this already.

Reply:
Interestingly, we had three similar methods before. Will be more convenient to have it one extension.